### PR TITLE
docs: updated readme to include for detail about running the tests

### PR DESCRIPTION
**Pull request by RoryT-U**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,64 @@
...
+- Anand Vannalath (avan0055@student.monash.edu)
+- Victor Huang (vhua006@student.monash.edu)
+- Sandy (ssen0027@student.monash.edu)
+- Snekith (skar0053@student.monash.edu)
+- Jovan Vales (jval0006@student.monash.edu)
+- Fathy Abdelshahid (fabd0009@student.monash.edu)
+- Rory Tobin-Underwood (rtob0002@student.monash.edu)
+
+## Developed Application
+The developed application is currently available [here](https://3170.fit/) to be enjoyed.
```
Comment (fuzzylogic123) [inline README.md:22]: Can't guarantee that I will continue hosting the backend after this unit is over though. So not sure if we want to put this here for handover.

File: README.md [hunk 2]
```diff
@@ -1,52 +1,64 @@
...
+To run the E2E tests locally, complete the following instructions.
+
+1. Start the server and client, as outlined in [Prepare for development](#Prepare-for-development)
+2. Move into the [tests](tests) directory of the project by typing the command `cd tests`.
+3. Install dependencies in [tests](tests) by running `npm install` in the terminal.
+4. Run the test by running `npm run test` in terminal.
+
+### Notes
+- Running `npm run test -- --headed` will create and display the tabs on the local device.
+- Occasionally the test will stall when a client browser fails to join, this can be fixed by manually clicking the join button.
```
Comment (fuzzylogic123) [inline README.md:60]: Are we not going to fix these bugs?

File: README.md [hunk 3]
```diff
@@ -1,52 +1,64 @@
...
 1. `npm run format`
 2. `npm run lint`
 
-# Testing
+## Testing
+To run the E2E tests locally, complete the following instructions.
+
+1. Start the server and client, as outlined in [Prepare for development](#Prepare-for-development)
+2. Move into the [tests](tests) directory of the project by typing the command `cd tests`.
+3. Install dependencies in [tests](tests) by running `npm install` in the terminal.
```
Comment (fuzzylogic123) [inline README.md:55]: Do we need to explain that they need to install the playwright browsers too? (The terminal will prompt you to install them)

**Review by fuzzylogic123 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RoryT-U) [reply README.md:22]: We can change it when that happens

File: README.md (hunk 2 above)
Comment (RoryT-U) [reply README.md:60]: Yeah but for now they still happen

File: README.md (hunk 3 above)
Comment (RoryT-U) [reply README.md:55]: Who knows they might need it.

**Pull request merged**